test/plugin/base_test.rb

```diff
@@ -1434,9 +1433,41 @@ def test_k8s_monitored_resource_fallback
           local_resource_id: RANDOM_LOCAL_RESOURCE_ID),
         # When 'kube-env' is present, "compute.googleapis.com/resource_name" is
         # not added.
-        expected_params: COMPUTE_PARAMS
-      },
-      # Specific cases for k8s_node.
+        expected_params: COMPUTE_PARAMS_NO_LOG_NAME
+      }
+    ].each do |test_params|
+      new_stub_context do
+        setup_gce_metadata_stubs
+        if test_params[:setup_metadata_agent_stub]
+          setup_metadata_agent_stubs
+        else
+          setup_no_metadata_agent_stubs
+        end
+        if test_params[:setup_k8s_stub]
+          setup_k8s_metadata_stubs
+        else
+          setup_no_k8s_metadata_stubs
+        end
+        setup_logging_stubs do
+          d = create_driver(test_params[:config], CONTAINER_TAG)
```

Comment:
The actual change is just
`          d = create_driver(test_params[:config], CONTAINER_TAG)`
for `k8s_container` and 
`          d = create_driver(test_params[:config])`
for `k8s_node`.

The test is too big so I split it.

Reply:
Since `k8s_node` doesn't care about the log tag, why not set its tag to `CONTAINER_TAG` as well?

Reply:
Because it's trying to simulate the behavior in Kubernetes. `CONTAINER_TAG` is not part of node logs.